system/conftest.py

```diff
@@ -37,6 +37,59 @@ def event_loop():
     await yield_((trustee_did, trustee_vk))
 
 
+@pytest.fixture()
+@async_generator
+async def initial_token_minting(pool_handler, wallet_handler, get_default_trustee):
```

Comment:
Yes, each function (test case) runs against new clear pool to avoid execution order dependencies and intermittent test failures due to initial unclear pool state.